src/components/TopMenu.tsx

```diff
@@ -183,6 +178,41 @@ function RewardsButton({
   ) : null
 }
 
+function MenuList() {
+  const { t } = useTranslation()
+  const { chainId } = useActiveWeb3React()
+  const gaugesAreActive = areGaugesActive(chainId)
+  const { pathname } = useLocation()
+  const activeTab = pathname.split("/")[1] as ActiveTabType
+  return (
+    <React.Fragment>
+      <NavMenu data-testid="swapNavLink" to="/" selected={activeTab === ""}>
```

Comment:
active tab supposed to be empty here?

Reply:
Yes.
Swap page's url is `https://saddle.exchange/#/`. and pathname.split("/") returns `""`.